packages/cascara/src/components/Table/0-Table.mdx

```diff
@@ -240,6 +253,16 @@ Specifies the attribute that uniquely identifies every object in the **data** ar
 >
 > If you see a warning like 'Each child in a list should have a unique "key" prop' from Table, please make sure you pass this prop to prevent Table's **auto-id-inference** from kicking-in and **possibly** infering **undefined** if the data you pass does not have any of the possible values.
 
+## sortable
+
+Specifies which sorting strategy to follow:
+
+| type          | example         | description                                           |
+| ------------- | --------------- | ----------------------------------------------------- |
+| Boolean       | true            | Specifies that all attributes are sortable            |
```

Comment:
I got confused in this documentation because of the capitalization of "eID" —i didn't want people to start writing code with that capitalization because in app_web and all Espressive the concept of "eid" is always lower case.
Then I read your code and realized `uniqueIdAttribute` really refers to unique-ID, and actually in the code values you use "eid" so it's probably better to clean the documentation and code comments to avoid replicating the pattern of writing "eID" when it's meant to be "eid".

Reply:
You have a good point, let us change that one.

Reply:
@odiseo42 this has been changed, thanks for the catch!